chapter_1/Functions/functions(2).py

```diff
@@ -0,0 +1,47 @@
+import socket
+
+
+def ret_banner(ip, port):
+    try:
+        socket.setdefaulttimeout(2)
+        s = socket.socket()
+        s.connect((ip, port))
+        banner = s.recv(1024)
+        return banner
+    except:
+        return
+
+
+def check_vulns(banner):
+    if 'FreeFloat Ftp Server (Version 1.00)' in banner:
+        print('{+} FreeFloat FTP Server is vulnerable.')
+    elif '3Com 3CDaemon FTP Server Version 2.0' in banner:
+        print('3Com 3CDaemon FTP Server is vulnerable.')
+    elif 'Ability Server 2.34' in banner:
+        print('{+} Ability FTP Server is vulnerable.')
+    elif 'Sami FTP Server 2.0.2' in banner:
+        print('{+} Sami FTP Server is vulnerable.')
+    else:
+        print('{-} FTP Server is not vulnerable.')
+    return
+
+
+def main():
```

Comment:
Please add newlines and fix the indentation of the lines in this function, like in the other file.

Reply:
Done.